packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/Abs_L1TokenGateway.sol

```diff
@@ -118,8 +118,9 @@ abstract contract Abs_L1TokenGateway is iOVM_L1TokenGateway, OVM_CrossDomainEnab
     function deposit(
         uint _amount
     )
-        public
+        external
```

Comment:
This feels much better, we should use slither to scan the codebase for this and fix more broadly.